Dockerfile

```diff
@@ -53,10 +53,11 @@ RUN mkdir -p /final/bin /final/share /final/libexec
 # 1. Build programs from source
 
 # Build RAxML
-# AVX should be widely-supported enough
+# linux/arm64 does not support -mavx and -msse3 compilation flags which are used in the official repository.
+# Make these changes in a fork for now.
+# TODO: Use the official repository if this PR is ever merged: https://github.com/stamatak/standard-RAxML/pull/50
 WORKDIR /build/RAxML
-RUN curl -fsSL https://api.github.com/repos/stamatak/standard-RAxML/tarball/v8.2.12 \
-  | tar xzvpf - --no-same-owner --strip-components=1 \
+RUN git clone --recurse-submodules --single-branch --branch simde https://github.com/nextstrain/standard-RAxML.git . \
```

Comment:
Done: 12758b7dfb588976fc36b95c2693179fd11253a9

This is also one less reliance on `git` to be aware of.

Reply:
Oops, that didn't work. The tarball endpoint creates a `simde` directory but doesn't provide its contents. I suppose this could work:

```sh
curl -fsSL https://api.github.com/repos/nextstrain/standard-RAxML/tarball/simde \
  | tar xzvpf - --no-same-owner --strip-components=1 \
 && curl -fsSL https://api.github.com/repos/simd-everywhere/simde-no-tests/tarball/294f44c9e948640f9e7d8a2ef7d6d196e14f26eb \
  | tar xzvpf - --no-same-owner --strip-components=1 -C simde \
 && make -f Makefile.AVX.PTHREADS.gcc \
 && cp -p raxmlHPC-PTHREADS-AVX /final/bin
```

but that's a bit too much repo-specific logic to me. Will revert back to `git clone` while taking your suggestions for shallow cloning.

Reply:
Ahh! I didn't expect/realize our `simde` branch introduced a new submodule, so I figured the submodules implied by `--recurse-submodules` were also bundled up in the previous tarball.

I guess we could vendor it in our `simde` branch instead of submodule-ing it, but also ok with reverting to `git clone` here.

Reply:
Actually, I'll make a couple changes here:

1. Add a commit to copy its contents into the `simde` branch, removing the need for submodules
2. Use the tarball endpoint with a commit hash instead of `simde` (and link to the branch in the comment above)

Reply:
Done: 6b305c74392c929ba20ba78d3cab21e0101f1ac6...258050a3e1ecfb438221d5f4f9c5f747ab89fb54